lib/contentful/migrationEntries.ts

```diff
@@ -6,7 +6,7 @@ import { ContentfulPartialOptions } from "../types"
 import { getClient } from "./client"
 import { MigrationEntry } from "./types"
 
-export async function createMigrationEntries(
+export async function createMigrationEntry(
```

Comment:
renamed since this function is returning a single entry and therefor the name is misleading